File: RecruiterLite/Models/Candidate.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RecruiterLite.Models;

public class Candidate: BaseEntity
{ 
    public string FirstName { get; set; }
    public string LastName { get; set; }
    [EmailAddress]
    public string Email { get; set; }
    public string PhoneNumber { get; set; }
    public string StreetAddress { get; set; }
    public string PostCode { get; set; }
    public string County { get; set; }
    public string Country { get; set; }
        
    [ForeignKey("CompanyId")]
    public int? CompanyId { get; set; }
}

File: RecruiterLite/ClientApp/src/app/models/Candidate.ts
export interface Candidate {
  id?: number
  firstName: string
  lastName: string
  fullName: string
  email: string
  phoneNumber: string
  streetAddress: string
  postCode: string
  county: string
  country: string
  companyId?: number
  companyName?: string
  isHiringManager: boolean
}
export interface PaginatedCandidates {
  pageIndex: number,
  pageSize: number,
  count: number,
  data: Candidate[];
}
